src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -59,10 +59,6 @@ class CompanyStep extends React.Component {
         this.defaultWebsite = lodashGet(props, 'user.isFromPublicDomain', false) ? 'https://' : `https://www.${Str.extractEmailDomain(props.session.email, '')}`;
     }
 
-    componentWillUnmount() {
-        BankAccounts.resetReimbursementAccount();
-    }
-
```

Comment:
Just found it while I was testing adding a vbba. Errors didn't show up on this step because of it. We used to do it for each step but when refactoring with the new forms, this one got left, but there is no reason to do it for this step more than for the other steps.